.dockerignore

```diff
@@ -1,4 +1,11 @@
-data/*
-!data/scores.json
+**/.git
```

Comment:
Previously, the `.git` folder was added to the docker image for the sake of building in a version number, namely a commit hash to display in the footer. The other ignores here leave out files that are irrelevant to the docker image build. Any time these files changed required the entire image to be rebuilt.